### ISSUE TITLE
진단 홈 - [잘 하고 있어요] 세부항목 클릭 시, 아래 내용 글 정렬 필요

### ISSUE THREAD

**Issue by doidoit** [finset-io/better-qa#88]
- 화면 위치
: 진단 홈  - [잘 하고 있어요/ 개선이 시급해요] - 세부 항목 클릭 시 나오는 글

- As is 
: 아래쪽에 있는 글 정렬 필요 
  (현재 글 양쪽 정렬이 통일되어 있지 않음. 개행되지 않아야 자연스러울 부분에 개행처리 되어 있음)

- To be
: 글 정렬 및 자연스러운 개행 처리 필요함. 

![Image](https://user-images.githubusercontent.com/98443099/179937256-d5442ff4-0c2d-4053-b35c-76d7ff579b8b.jpg)
![Image](https://user-images.githubusercontent.com/98443099/179937255-5187d444-4e3b-4d25-8d5d-fab57d2c995f.jpg)

**Comment by hanakim00** [comment]
해당내용으로 전체 페이지 전수검사 필요합니다. (디자인에서 강제 줄바꿈되어있는 레이아웃 수정 작업)

**Comment by jholee7** [comment]
보통 P2면 수정하는데 시간이 얼마나 소요 되나요? 가이드를 주실 수 있으시겠습니까?

**Comment by kimpilwon** [comment]
해당 내용은 인텔리코드측에서 수정해야할 qa라서 물어봐야 일정이 나올것같은데 차주안엔 될것같습니다

**Comment by jholee7** [comment]
네, 핏프로님 답변 감사합니다~ 다른 QA들도 대략적인 일정 등이 지정되거나 협의되서 이야기 될 수 있도록 하면 저희 전체적인 속도를 올릴 수 있을 것이라고 생각합니다. 한번 고려해봐 주세요.
